src/main/java/kevesse_kokanyolo_kod/menus/Printer.java

```diff
@@ -1,12 +1,57 @@
 package kevesse_kokanyolo_kod.menus;
 
 
-public interface Printer {
-    public default void println(String msg) {
+import java.io.*;
+import java.util.Scanner;
 
+public class Printer {
+    public static FileWriter fileWriter = null;
+    public static Scanner scanner = null;
+    private static int indentCounter = 0;
+
+    public Printer(String inputFileName, String outputFileName) throws IOException {
+        fileWriter = new FileWriter(outputFileName);
+        scanner = new Scanner(new File(inputFileName));
+    }
+    public Printer() {
+        fileWriter=null;
+        scanner = new Scanner(System.in);
+    }
+
+    public void println(String msg) {
+        try {
+            for (int i = 0; i < indentCounter - 1; i++) {
+                if (fileWriter!=null) fileWriter.write("\t");
+                else System.out.print("\t");
+            }
+            if (fileWriter!=null) fileWriter.write(" " + msg + "\n");
+            else System.out.println(msg);
+        } catch (IOException e) {
+            System.out.println("Nem lehet a fájlba írni!");
+            System.exit(1);
+        }
     };
-    public default void print(String msg) {
-    
-    };
+    public void printWithIndents(String msg) {
```

Comment:
Ez mire való? a println() is indentálva printel, nem kell ez a metódus, szerintem csak azért csináltad, mert "majd jól jöhet". ha ilyenre szükség van akkor inkább a printernek adjunk egy disableIndents() metódust.

Reply:
A szkeleton refactorálásához gondoltam, hogy szükség lesz majd rá. A disableIndendts() funkciója mi lenne? Ha nem szkeleton van futtatva akkor ki lehesesn kapcsolni?